src/training/validator.cpp

```diff
@@ -353,6 +355,12 @@ float TranslationValidator::validate(const std::vector<Ptr<ExpressionGraph>>& gr
 
   if(options_->hasAndNotEmpty("valid-translation-output")) {
     fileName = options_->get<std::string>("valid-translation-output");
+    fileName = fmt::format(fileName.c_str(),
```

Comment:
I know that fmt is included in spdlog, but maybe we can add an explicit `include` at the beginning of the file for clearness?

Reply:
I put in a comment in the affected function.